CHANGELOG.rst

```diff
@@ -2,10 +2,20 @@
 ======
 NOT RELEASED
 
+Features
+--------
+* Add Cluster ssl_context option to enable SSL (PYTHON-995)
+* Allow encrypted private keys for 2-way SSL cluster connections (PYTHON-995)
+
 Other
 -----
 * Fail faster on incorrect lz4 import (PYTHON-1042)
 
+Deprecations
+------------
+
+* Using Cluster.ssl_options to enable SSL is deprecated and will be removed in
+  the next major release, use ssl_context.
```

Comment:
RFC: I don't know what you think of a this section in our CHANGELOG... but I've seen it in another project and I really like that. It makes it clear for users if they have important deprecation tasks to fix in their code. Note that there is no migration guide for minor releases.

Reply:
Well, nobody checks the migration guide until they migrate to a major version. Note that this is not a migration guide replacement. This is to document changes so they can fix/be aware along the way of minor versions. That also means, no more surprises DeprecatedWarning showing in the logs. It's also useful to see everything to recheck when writing/finalizing the migration guide. Let me know what you think.

Reply:
Ah, I see the difference. That sounds great, +1.